src/plugins/intel_gpu/src/graph/graph_optimizer/reorder_inputs.cpp

```diff
@@ -372,8 +372,52 @@ void minimize_local_reorders(program& p, std::map<program_node*, format::type>&
     }
 }
 
+static format get_target_output_format(layout_optimizer& lo, const std::map<program_node*, format::type>& fmt_map, program_node *node) {
+    // 1. Check required_output
+    if (lo.get_optimization_attributes().use_onednn_impls) {
+        // If onednn is not used, need to ignore get_required_layout result as it is from onednn
+        auto ret = node->get_required_output();
```

Comment:
I'd like to ask the intention or future provision of this flag / pass regarding "required' layouts. 
In this comment, it is mentioned that the required_layout is  for onednn. 
If this is a concrete decision for the future too, isnt it better to set the flag's name as "onednn_layout" to reduce the future confusion?

Reply:
In the future, hope to generalize `get_required_layout` to cldnn case as well. The comment about "onednn" is because it is supported only from ondnn at this moment.